### PR TITLE
Analytics: Add option to pass destSDKBaseURL to rudderstack load method

### PR DESCRIPTION
**Pull request by gassiss**
**What is this feature?**

Adds an option to pass `destSDKBaseURL` to the Rudderstack SDK. 

**Why do we need this feature?**

When loading the Rudderstack SDK version 1.1 or newer, this option can be passed to override the URL used to load SDK

**Who is this feature for?**

Any person that wants to override this option, and is using Rudderstack SDK version 1.1 or newer.

**Which issue(s) does this PR fix?**:

N/A

**Special notes for your reviewer:**

Please check that:
- [X] It works as expected from a user's perspective.
- [X] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -575,6 +575,12 @@ URL to load the Rudderstack SDK.
 Optional. If tracking with Rudderstack is enabled, you can provide a custom
 URL to load the Rudderstack config.
 
+### rudderstack_integrations_url
+
+Optional. If tracking with Rudderstack is enabled, you can provide a custom
+URL to load the SDK for destinations running in device mode. Only valid for
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:581]:
```suggestion
URL to load the SDK for destinations running in device mode. This setting is only valid for
```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 2]
```diff
@@ -575,6 +575,12 @@ URL to load the Rudderstack SDK.
...
+Rudderstack version 1.1 or newer.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:582]:
```suggestion
Rudderstack version 1.1 and higher.
```

**Review by chri2547 — COMMENTED** [review]
Added a couple of suggestions. Docs approved. Thanks for the contribution!

**Review by chri2547 — COMMENTED** [review]
Docs are approved. Thank you.

**Review by joshhunt — APPROVED** [review]

**Pull request merged**